datasets/ONS-foreign-direct-investment-involving-uk-companies/main.py

```diff
@@ -265,24 +273,36 @@ def split_overrides(bag, splits):
     "gulf-arabian": "gulf-arabian-countries",
     "other-asian": "other-asian-countries"
 }
-observations["FDI Area"] = observations["FDI Area"].map(lambda x: fix.get(x, x))
+observations['FDI Area'] = observations['FDI Area'].map(lambda x: fix.get(x, x))
 
 
 # +
-
-inward_scraper.dataset.title = inward_scraper.dataset.title.replace(': inward', '')
-inward_scraper.dataset.comment = inward_scraper.dataset.comment.replace(
-    'into the UK', 'into the UK and of UK companies abroad')
-inward_scraper.dataset.landingPage = landingPages
-
-from gssutils.metadata import THEME
-inward_scraper.dataset.theme = THEME['business-industry-trade-energy']
-inward_scraper.dataset.family = 'trade'
-
-cubes.add_cube(inward_scraper, observations.drop_duplicates(), inward_scraper.dataset.title)
-cubes.output_all()
-
-# -
-trace.render("spec_v1.html")
-
-
+observations.to_csv('foreign_direct_investment-observations.csv', index = False)
+
+catalog_metadata: CatalogMetadata = CatalogMetadata(
+    title = "Foreign direct investment involving UK companies (directional): inward",
```

Comment:
Title should not have inward at the end since the final dataframe includes inward and outward data. 

Title should be : Foreign direct investment involving UK companies (directional)

Reply:
Title changed to:- Foreign direct investment involving UK companies (directional)